### DISCUSSION TITLE
Welcome PrepBoost Discussions!!

### DISCUSSION THREAD

**Discussion by bhargavbvs** [bhargavbvs/PrepBoost#47] [General]
Hey guys,

Thanks for coming to our discussions page. 

PrepBoost is an Edtech platform designed for students preparing for exams conducted by UPSC (Union Public Service Commission) of India. 

- You can share any of your thoughts about the website.
- Build the community docs together.
- Engage with other members to connect.
- We will share any future updates about the website and new features here.

To get started, comment an introduction of yourself and about your future goals.

**Comment by srisai1234** [comment]
Hello everyone,

- This website particularly helps UPSC aspirants which is the toughest exam in India.
- Till now we have developed navigation bar and components in that in frontend part.
- In the components we have developed pages for each component such as year wise and topic wise questionsand home page.
- This gives access to different questions for easier preparation of users.
- In future we will be adding questions to these pages and making our website more effective.
